src/mutable_data.rs

```diff
@@ -259,7 +268,7 @@ macro_rules! impl_mutable_data {
                 Ok(())
             }
 
-            /// Delete permissions for the provided user.
+            /// Deletes permissions for the provided user.
```

Comment:
We should mention what the version parameter means and that it must be equal to current version + 1.